README_JP6.md

```diff
@@ -115,8 +115,9 @@ Following steps required:
     1.	Configure Jetson AGX CSI Connector
     2.	Configure for compatible hardware
```

Comment:
these steps doesn't mention about updating the `Image` file like in other readmes (JP4 / JP5). Is updating the kernel not required for JP6?

Reply:
JP6 not required.